assets/filter-hub-tags.js

```diff
@@ -4,7 +4,7 @@ var displayCount = Number(filterScript.attr("display-count"));
 var pagination = filterScript.attr("pagination");
 
 var options = {
-  valueNames: ["github-stars-count-whole-number", { data: ["tags", "date-added"] }],
+  valueNames: ["github-stars-count-whole-number", { data: ["tags", "date-added"] }, { data: ["tags", "title"] }],
```

Comment:
Instead of adding a second object (`{ data: ["tags", "title"] }`) can you try adding "title" to the same array as "date-added"? I added "date-added" to that array, so I think adding another data element should work.

Reply:
Yep, that works! I'll update the PR